nucliadb/nucliadb/ingest/chitchat.py

```diff
@@ -135,3 +128,20 @@ async def socket_reader(
     async def close(self):
         self.chitchat_update_srv.close()
         self.task.cancel()
+
+
+def build_member_from_json(x):
```

Comment:
Can you provide proper parameter name, on lambda makes sense but on function should be semantic naming